Return null None from get_or_fetch_member upon unfound members & bad requests (#169)
python-discord/bot-core: docs/changelog.rst

Changelog
=========

- :release:`9.3.0 <13th December 2022>`

> - :feature:`169` Return None upon sending a bad request to fetch a member

Reviewer: ```suggestion
- :feature:`169` Return :obj:`None` upon receiving a bad request from Discord in :obj:`pydis_core.utils.members.get_or_fetch_member`
```
By doing :obj:`foo` you can link to the actual objects. This change rendered looks like
![image](https://user-images.githubusercontent.com/9753350/207298310-af653970-dd7d-48bb-8908-62791cb44b00.png)

Author: Neat ! I didn't know that.
Applied in 5a2f70199cf9e9bbd10c4137b72d8fdec9598ce0, thanks !
